build-scripts/windows-buildbot-zipper.bat

```diff
@@ -13,16 +13,10 @@ IF EXIST zipper GOTO zippermade
 mkdir zipper
 :zippermade
 mkdir zipper\SDL
-mkdir zipper\SDL\include

```

Comment:
Depending on how @icculus plans to reintegrate the buildbot, we may just need a new batch file entirely for the Actions uploads.